nvflare/private/fed/server/server_engine.py

```diff
@@ -214,6 +214,22 @@ def start_app_on_server(self, run_number: str, job_id: str = None, job_clients=N
             self.engine_info.status = MachineStatus.STARTED
             return ""
 
+    def _recv(self, conn, job_id, run_number):
+        try:
+            return conn.recv()
+        except BaseException as e:
+            self.logger.error(f"Received unexpected reply from Client for Job: {job_id}, run_number: {run_number}: {e}")
```

Comment:
I think this is confusing.  conn is localhost for IPC.  In server, that's server child process.  @yhwen , can you help clarify this?

Reply:
That's correct. This communication is between the parent and client process on the same server. Not between the client and server. When the exception happens, we already logged the message showing the communication and let it try again.

The original issue caused the error is because the child process got terminated. We need to investigate that root caused the child process crash.

Reply:
Ok, I can change the wording, of the message, but we need to log the error at where it happens. so we know what's going on.

Reply:
I update the message, please check again @IsaacYangSLA @yhwen